src/components/common/ConversationsComponent.js

```diff
@@ -0,0 +1,43 @@
+import React, {PropTypes} from 'react';
+import * as C from '../../res/strings/strings-en';
+import imgCount from '../../res/images/common/yellow-circle.png';
+// IDEA: http://www.bypeople.com/sliding-css-chat-bubbles/ nice page
+export const ConversationsComponent = (props) => {
```

Comment:
I see that only use the `conversations` prop, so you can do this trick to clean the code a little bit:
`export const ConversationsComponent = ({ conversations }) => {`

Then you not need to user `props.conversations` just `conversations` will work